policyengine_canada/variables/household/person/care_expenses.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class care_expenses(Variable):
+    value_type = float
+    entity = Person
+    label = "Amount of childcare expenses incurred by the child"
```

Comment:
So we can include elder and disability care in it
```suggestion
    label = "Amount of care expenses incurred by the person"
```